chart/templates/webserver/webserver-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
   strategy:
     type: RollingUpdate
     rollingUpdate:
-      maxUnavailable: 1
+{{ toYaml .Values.webserver.rollingUpdate | indent 6 }}
```

Comment:
maybe more straight forward to have `maxUnavailable: {{ .Values.webserver.rollingUpdate.maxUnavailable }}` here. Let me know if you have different opinion.

Reply:
I think, It may be better to support both `maxSurge` and `maxUnavailable`.

https://github.com/apache/airflow/pull/13906/files#diff-a9d07052701ab3b718c82ac0fe74b965a8260b06bb5906362081718b0ee593b4R419

Reply:
Like you said, `maxUnavailable` maybe more straight forward, but the PR title is "make webserver rolling update", not just `maxUnavailable`.

Reply:
As I mentioned above, the change you intend to make here is no longer "make webserver rolling update", instead, it's allowing more configuraitons.

As already suggested, please update the PR title and description; or create a new PR accordingly.

Reply:
@XD-DENG Well.. I modified the title.

Reply:
Do you mind addressing the comment above? I don't see it necessary to use `toYaml` here.

> maybe more straight forward to have `maxUnavailable: {{ .Values.webserver.rollingUpdate.maxUnavailable }}` here. Let me know if you have different opinion.

In addition, please also update the commit subject and add description properly (that's what appear eventually in the commit history)

Reply:
Rolling Update of kubernetes not just `maxUnavailable`

>I think, It may be better to support both maxSurge and maxUnavailable.
> https://github.com/apache/airflow/pull/13906/files#diff-a9d07052701ab3b718c82ac0fe74b965a8260b06bb5906362081718b0ee593b4R419

Reply:
Then where is your `maxSurge` in `values.yaml`?  :-)
As well as the necessary update in `chart/README.md`? 

Please make *all* necessary changes and rebase.

Reply:
`rollingUpdate.maxSurge` is an optional field, default value is 25%, I added the comment of maxsurge.